elizabeth/data/kz/text.json

```diff
@@ -0,0 +1,145 @@
+{
+  "alphabet": {
+	"uppercase": [
+	  "А",
+	  "Ә",
+	  "Б",
+	  "В",
+	  "Г",
+	  "Ғ",
+	  "Д",
+	  "Е",
+	  "Ё",
+	  "Ж",
+	  "З",
+	  "И",
+	  "Й",
+	  "К",
+	  "Қ",
+	  "Л",
+	  "М",
+	  "Н",
+	  "Ң",
+	  "О",
+	  "Ө",
+	  "П",
+	  "Р",
+	  "С",
+	  "Т",
+	  "У",
+	  "Ұ",
+	  "Ү",
+	  "Ф",
+	  "Х",
+	  "Һ",
+	  "Ц",
+	  "Ч",
+	  "Ш",
+	  "Щ",
+	  "Ъ",
+	  "Ы",
+	  "І",
+	  "Ь",
+	  "Э",
+	  "Ю",
+	  "Я"
+	],
+	"lowercase": [
+	  "а",
+	  "ә",
+	  "б",
+	  "в",
+	  "г",
+	  "ғ",
+	  "д",
+	  "е",
+	  "ё",
+	  "ж",
+	  "з",
+	  "и",
+	  "й",
+	  "к",
+	  "қ",
+	  "л",
+	  "м",
+	  "н",
+	  "ң",
+	  "о",
+	  "ө",
+	  "п",
+	  "р",
+	  "с",
+	  "т",
+	  "у",
+	  "ұ",
+	  "ү",
+	  "ф",
+	  "х",
+	  "һ",
+	  "ц",
+	  "ч",
+	  "ш",
+	  "щ",
+	  "ъ",
+	  "ы",
+	  "і",
+	  "ь",
+	  "э",
+	  "ю",
+	  "я"
+	]
+  },
+  "answers": [
+	"Иә",
+	"Жоқ",
+	"Мүмкін"
+  ],
+  "color": [
+	"Ақ",
+	"Жасыл",
+	"Көгілдір",
+	"Көк",
+	"Күлгін",
+	"Қара",
+	"Қызғылт сары",
+	"Қызыл",
+	"Сары",
+	"Сұр"
+  ],
+  "level": [
+	"төмен",
+	"орташа",
+	"жоғары",
+	"өте жоғары",
+	"төтенше",
+	"сыни"
+  ],
+  "quotes": [
+
+  ],
+  "text": [
+
+  ],
+  "words": {
+	"normal": [
+
+	],
+	"bad": [
+
+	]
+  },
+  "sentence": {
```

Comment:
@bbb1991 This is only  for RussianSpecProvider. If you have not planes to add special provider for `kz`, please remove it (i mean `"sentence": [...]` )